### PR TITLE
Release `v4.5.0`

### PR DESCRIPTION
**Pull request by sphuber**

### CODE REVIEW

**Comment by sphuber** [comment]
@mbercx nudge nudge

**Comment by mbercx** [comment]
Apologies @sphuber! I guess my first comment is that the changelog doesn't seem to be using the conventional sections we defined in the [`update_changelog.py`](https://github.com/aiidateam/aiida-quantumespresso/blob/main/.github/workflows/update_changelog.py) script?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
+## v4.5.0
+
+### ✨ New features
+- `PwBaseWorkChain`: new handler for BFGS history failure [[0224f8a]](https://github.com//commit/0224f8a4cf8122a916e2b2c5be11f3a8a811f740)
```
Comment (mbercx) [inline CHANGELOG.md:4]: Seems the links are incomplete?

```suggestion
- `PwBaseWorkChain`: new handler for BFGS history failure [[0224f8a]](https://github.com/aiidateam/aiida-quantumespresso/commit/0224f8a4cf8122a916e2b2c5be11f3a8a811f740)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+- `PhBaseWorkChain`: allow generation of q-point mesh via `qpoints_distance` [[b9c7517]](https://github.com//commit/b9c7517744e645a93d4afc9b1999881fc39a0e46)
+- Support calculating the XPS spectra of the atoms specific by indices [[fc1a940]](https://github.com//commit/fc1a940d4a60f22b42ec0a069a06436c0c9ae0f5)
+
+### 🐛 Bug fixes
+- `PdosWorkChain`: Fix constrained magnetization case [[a68e1e1]](https://github.com//commit/a68e1e15c11f6ad4461921145c648d75c49ff26c)
+- Fix missing `max_iterations` in overrides [[9061ea5]](https://github.com//commit/9061ea5df65e4fe95f77309d9abb5a3c7f64bb9f)
+- Improve `PhBaseWorkChain` overrides/protocol [[39287e0]](https://github.com//commit/39287e03cb6bbf1915662685a5c441e9c7c36030)
```
Comment (mbercx) [inline CHANGELOG.md:11]: Isn't this more of a new feature or improvement?

**Review by mbercx — CHANGES_REQUESTED** [review]
Just two second comments. ^^

**Review by mbercx — APPROVED** [review]
LGTM! 🚀

**Pull request merged**